unitTests/testStackTrace.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, Lawrence Livermore National Security, LLC and LvArray contributors.
+ * All rights reserved.
+ * See the LICENSE file for details.
+ * SPDX-License-Identifier: (BSD-3-Clause)
+ */
+
+// Source includes
+#include "system.hpp"
+#include "Macros.hpp"
+
+// TPL includes
+#include <RAJA/RAJA.hpp>
+#include <gtest/gtest.h>
+
+
+namespace LvArray
+{
+namespace testing
+{
+
+std::string __attribute__((noinline)) baz()
+{
+  return system::stackTrace( true );
+}
+
+std::string __attribute__((noinline)) bar()
+{
+  return baz();
+}
+
+std::string __attribute__((noinline)) foo()
+{
+  bar();
+  std::string const result = bar();
+  bar();
+  return result;
+}
+
+#if defined( LVARRAY_USE_OPENMP )
+
+TEST( stackTrace, OpenMP )
```

Comment:
No way, this is for sure host only code. The reason I have the `OpenMP` test is because with clang10 you'd get a segfault in the first call to `backtrace` if you "recently" called some threaded code. If you had a previous call to `stacktrace` or OpenMP were sufficiently removed from a threaded region it would succeed.

Reply:
I should have been clearer. I meant does this work above a CUDA kernel? I.e. if the kernel aborts, then what do we get? Nothing? Line number of the kernel launch?

Reply:
It appears that at least `LVARRAY_ERROR` causes RAJA (in release) or CUDA (in debug) to raise an exception which doesn't lead trigger our signal handler. Even if it did trigger the handler it would only happen where the exception turned fatal (propogated out of main or a noexcept function) so the stacktrace wouldn't be super helpful. We could probably fix this by wrapping `RAJA::forall` with a `try` statement but that's for whatever PR adds exceptions to GEOSX.